data/overlays/gamestates/z_file_nameset_PAL.data.s

```diff
@@ -129,6 +129,9 @@ glabel D_80812770
 .word func_8080BE30
 glabel D_80812814
  .word 0x004600C8
+
+# z_file_choose
```

Comment:
What do you mean by that comment? If this was a file boundary it would be 16 bytes aligned like usual, so if that's what you meant it can't be correct.